### PR TITLE
Gibs takedown take two

### PR DESCRIPTION
**Pull request by cvalarida**
Let's try this again...But better this time.

image

### CODE REVIEW

**Review by markgreenburg — APPROVED** [review]
Also LGTM ;)

**Review by omgitsbillryan — APPROVED** [review]
LGTM.  I'm glad we basically already had support for this, I think I'll close the feature request I had just opened for this: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/8834

Secondly, what was the point of `maintenance_line2`?

**Review by kreek — APPROVED** [review]

**Comment by cvalarida** [comment]
@omgitsbillryan The second line shows a smaller text (h4, I think) underneath the h3 shown.
Whomever gets to it first works for me. May the odds be ever in your favor.

...except it might fail in the e2e because of the maintenance page, so...We'll see how that goes. If it does, I'll just disable the test for now and try again. :grimacing:

**Comment by cvalarida** [comment]
Eh, that's what I was afraid of. Just pushed up a fix to bypass the tests for the time being. Tried it locally to confirm it works, but I think my local e2e tests are just broken right now (newly-installed dependencies and all that jazz that I haven't tried yet).

File: content/pages/education/gi-bill/post-9-11/ch-33-benefit.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ entryname: post-911-gib-status
 layout: page-react.html
 includeBreadcrumbs: true
 in_maintenance: true
-maintenance_line1: We’re sorry, our system is temporarily down while we fix a few things. Please try again later.
+maintenance_line1: We’re sorry. Our system is temporarily down while we fix a few things. Please try again later.
```
Comment (omgitsbillryan) [inline content/pages/education/gi-bill/post-9-11/ch-33-benefit.md]: Really?

Comment (cvalarida) [reply content/pages/education/gi-bill/post-9-11/ch-33-benefit.md]: I just work here. :man_shrugging:

**Pull request merged**